### PR TITLE
Issue #9 all packages require pcsc-lite daemon and library

### PR DESCRIPTION
**Pull request by articice**
https://github.com/open-eid/gentoo/issues/9

### CODE REVIEW

File: app-crypt/qdigidoc/qdigidoc-3.12.2.ebuild [hunk 1]
```diff
@@ -19,7 +19,7 @@ EGIT_COMMIT="v${PV}"
 #endif
 
 RDEPEND="dev-libs/openssl:=
-	>=dev-libs/opensc-0.14
+	>=dev-libs/opensc-0.14[pcsc-lite]
```
Comment (martinpaljak) [inline app-crypt/qdigidoc/qdigidoc-3.12.2.ebuild:22]: Isn't there a newer version of OpenSC available (0.16) ?

Comment (urmet) [reply app-crypt/qdigidoc/qdigidoc-3.12.2.ebuild:22]: ~opensc 0.13 to 0.16 are all \~arch masked. unless 0.12 is good enough, might as well put 0.16 here. or what is the actual needed version?~
E:
0.14 comes from #7 - so it seems to be correct for now.

Comment (metsma) [reply app-crypt/qdigidoc/qdigidoc-3.12.2.ebuild:22]: EstEID 3.5 smartcard support is since 0.15.0

Comment (urmet) [reply app-crypt/qdigidoc/qdigidoc-3.12.2.ebuild:22]: opensc changelog claims 0.14 http://github.com/OpenSC/OpenSC/blob/master/NEWS#L148

Comment (metsma) [reply app-crypt/qdigidoc/qdigidoc-3.12.2.ebuild:22]: OK my bad

**Comment by articice** [comment]
@martinpaljak well, one of the plugins (esteid-firefox-plugin) doesn't work without the other - firefox-pkcs11-loader, which is an add-on btw.
That's why the runtime dependency.

Feel free to add a metapackage that pulls both.

**Comment by martinpaljak** [comment]
If that is the case, then there is a bug somewhere. The PKCS#11 loader plugin is only necessary to "automate" the loading of the PKCS#11 module for SSL authentication.

**Comment by articice** [comment]
According to this doc, http://id.ee/index.php?id=34392
and my experience, it is impossible to authenticate at any website without pkcs loader addon, which renders the esteid-firefox-plugin useless.

**Comment by urmet** [comment]
It actually works without the loader. You just have to do the old-school method and manually add /usr/lib/onepin-opensc-pkcs11.so as a security device. http://i.imgur.com/uO8zYKD.png

**Comment by articice** [comment]
Ah, didn't know that. Well, maybe a use flag that controlls this dependency then? An inverse one, that removes the dependency when set, like 'pkcs11', 'opensc' or 'opensc-ocsp', or 'no-addon' or 'oldschool', etc.

And if the addon is not pulled in by dependency, a message that suggests adding /usr/lib/onepin-opensc-pkcs11.so to firefox config?

It's better when it all works just after merging, right?

**Comment by urmet** [comment]
I would call the use flag something like 'autoloader', default it to off and add a message. Inverse flags just create confusion.

**Comment by articice** [comment]
Ok, all done.
Please pull.

**Comment by urmet** [comment]
You actually have to define the use flag before using it. I'll fix it in the next commit.

**Pull request merged**